google-cloud-spanner/lib/google/cloud/spanner/database/backup_info.rb

```diff
@@ -84,14 +84,23 @@ def source_database_path
           end
 
           ##
-          # The backup contains an externally consistent copy of
-          # `source_database` at the timestamp specified by `create_time`.
-          # received.
+          # The timestamp indicating the creation of the backup.
           # @return [Time]
           def create_time
             Convert.timestamp_to_time @grpc.create_time
           end
 
+          ##
+          # The backup contains an externally consistent copy of
+          # `source_database` at the timestamp specified by
+          # the `version_time` received. If no `version_time` was
+          # given during the creation of the backup, the `version_time`
+          # will be the same as the `create_time`.
+          # @return [Time]
+          def version_time
+            Convert.timestamp_to_time @grpc.version_time unless @grpc.version_time.nil?
```

Comment:
Just use `Convert.timestamp_to_time @grpc.version_time`. Convert is already handling nil value.

Reply:
Fixed.